deps/exokit-bindings/webaudiocontext/src/AudioBuffer.cpp

```diff
@@ -165,7 +165,6 @@ NAN_METHOD(AudioBuffer::CopyToChannel) {
 
 AudioBufferSourceNode::AudioBufferSourceNode() {
   audioNode.reset(new lab::FinishableSourceNode([this](lab::ContextRenderLock &r){
-    QueueOnMainThread(r, std::bind(ProcessInMainThread, this));
```

Comment:
I think this was used to implement [`onended`](https://developer.mozilla.org/en-US/docs/Web/API/AudioScheduledSourceNode/onended) or such. Is that still firing?

Reply:
It is now.